src/backend/partaj/core/api/user.py

```diff
@@ -10,17 +10,21 @@
 from rest_framework.permissions import IsAuthenticated
 from rest_framework.response import Response
 
+from ..indexers import UsersIndexer
 from ..serializers import UserLiteSerializer, UserSerializer
+from .common import AutocompleteMixin, ViewSetMetadata
 from .permissions import NotAllowed
 
 User = get_user_model()
 
 
-class UserViewSet(viewsets.ReadOnlyModelViewSet):
+class UserViewSet(AutocompleteMixin, viewsets.ReadOnlyModelViewSet):
```

Comment:
Is it needed ? As you already add an autocomplete for your user_lite endpoint ?

Reply:
Right, I forgot to remove that 👍